packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx

```diff
@@ -25,13 +27,14 @@ export const AccordionContent: React.FunctionComponent<AccordionContentProps> =
   isHidden = false,
   isFixed = false,
   'aria-label': ariaLabel = '',
+  container,
```

Comment:
can we call this `component` like we do in other components like button.

Reply:
Done.